xcode/Subconscious/Shared/Components/AddressBook/FollowUserView.swift

```diff
@@ -15,6 +16,15 @@ struct FollowUserView: View {
     }
     var send: (AddressBookAction) -> Void
     
+    func onQRCodeScanResult(res: Result<ScanResult, ScanError>) {
```

Comment:
Non-blocking: we could send `res` through a `.qrCodeResult(res)` action and do this in the update function instead of in the view.